src/axom/quest/SignedDistance.hpp

```diff
@@ -52,7 +53,18 @@ struct UcdMeshData
   /*!
    * \brief Returns the type of the cell at the given index.
    */
-  AXOM_HOST_DEVICE mint::CellType getCellType(IndexType cellId) const;
+  AXOM_HOST_DEVICE mint::CellType getCellType(IndexType cellId) const {
```

Comment:
Please run this through `make style`

Reply:
hmm, I'm not actually sure how to do this from inside our serac build system, I get
```
sam@provolone:~/code/serac/build/axom$ make style
make: *** No rule to make target 'style'.  Stop.
```
Is the `style` target only optionally enabled?

Reply:
Your config needs a `CLANGFORMAT_EXECUTABLE` variable to use `make style`
(and I think this needs to be clang@10)

Something similar to https://github.com/LLNL/axom/blob/5f7869f21c4a2ee0ab0756e6f241690db6bcde86/host-configs/rzgenie-toss_3_x86_64_ib-clang%4010.0.0.cmake#L98)

I think @white238 should be able to help you set this up w/ serac

Reply:
I just handled it.  If this becomes a normal thing, I will reinvestigate a better way of handling styling.  Clangformat is just so fickle on versions.  I turned off the style checks on axom in serac so that it didnt constantly touch files.

Made an issue so I don't forget: https://github.com/LLNL/serac/issues/633